aws/variables.tf

```diff
@@ -0,0 +1,46 @@
+variable "region" {}
+
+variable "cluster_name" {}
+
+variable "map_roles" {
+  description = "Additional IAM roles to add to the aws-auth configmap."
+
+  type = list(object({
+    rolearn  = string
+    username = string
+    groups   = list(string)
+  }))
+
+  default = []
+}
+
+variable "min_capacity" {
+  description = "Min number of workers"
+  default = 2
+}
+
+variable "cidr" {
+  description = "The CIDR block for the VPC. Default value is a valid CIDR, but not acceptable by AWS and should be overridden"
+
+  default = "172.16.0.0/16"
+}
+
+variable "private_subnets" {
+  description = "List of private subnets inside the VPC"
+
+  default = ["172.16.0.0/21", "172.16.16.0/21", "172.16.32.0/21", "172.16.48.0/21", "172.16.64.0/21"]
+}
+
+variable "public_subnets" {
+  description = "List of public subnets inside the VPC"
+
+  default = ["172.16.8.0/22", "172.16.24.0/22", "172.16.40.0/22", "172.16.56.0/22", "172.16.72.0/22"]
+}
```

Comment:
those are the same as written in `vpc` variables file.
We should remove the duplication or at least share it somehow.

Reply:
TF doesn't know to take default from modules 
So - if someone will use this specific or from full blown he needs it